src/Sema.zig

```diff
@@ -23332,7 +23352,19 @@ fn explainWhyTypeIsNotExtern(
         .Frame,
         => return,
 
-        .Pointer => try mod.errNoteNonLazy(src_loc, msg, "slices have no guaranteed in-memory representation", .{}),
+        .Pointer => {
+            const pointee_ty = ty.elemType2();
+            if (pointee_ty.zigTypeTag() == .Pointer) {
+                try sema.explainWhyTypeIsNotExtern(block, src, msg, src_loc, pointee_ty, position);
+            } else {
+                if (ty.isSlice()) {
+                    try mod.errNoteNonLazy(src_loc, msg, "slices have no guaranteed in-memory representation", .{});
+                } else {
+                    try mod.errNoteNonLazy(src_loc, msg, "pointer to comptime-only type '{}'", .{pointee_ty.fmt(sema.mod)});
+                    try sema.explainWhyTypeIsComptime(block, src, msg, src_loc, pointee_ty);
```

Comment:
`explainWhyTypeIsComptime` only passes `block` and `src` to itself so they should be removed.

Reply:
But their signature is 
```zig
fn explainWhyTypeIsComptime(
    sema: *Sema,
    block: *Block,
    src: LazySrcLoc,
    msg: *Module.ErrorMsg,
    src_loc: Module.SrcLoc,
    ty: Type,
) CompileError!void
```
Is it ok to omit explanation?

Reply:
`explainWhyTypeIsComptime` doesn't use them it only passes them recursively to itself.